src/openrct2-ui/interface/ViewportInteraction.cpp

```diff
@@ -527,14 +527,14 @@ static void viewport_interaction_remove_footpath(TileElement* tileElement, int32
     if (w != nullptr)
         footpath_provisional_update();
 
-    tileElement2 = map_get_first_element_at(x / 32, y / 32);
+    tileElement2 = map_get_first_element_at(mapCoords.x / 32, mapCoords.y / 32);
```

Comment:
is there not an overload that takes `CoordsXY`

Reply:
There is not, just an overload for `map_get_surface_element_at`